README.md

```diff
@@ -59,6 +60,9 @@ php.ini parameters for shadow. Default is fine for most cases.
 | shadow.mkdir\_mask | 0755    | Mask used when creating new directories on instances |
 | shadow.debug       | 0       | Debug level (bitmask)                                |
 | shadow.cache\_size | 10000   | Shadow cache size (in bytes, per process)            |
+| shadow.override    | See bundled shadow.ini | Internal functions overridden by shadow |
```

Comment:
This is now inconsistent with the renamed shadow.ini → zz-shadow.ini. Not sure, however, if we need to enforce it this way.